packer/install-dependencies.sh

```diff
@@ -40,7 +40,7 @@ for CNI_PLUGIN_VERSION in $CNI_PLUGIN_VERSIONS; do
     downloadCNI
 done
 
-for CONTAINERD_VERSION in $CONTAINERD_VERSIONS; do
+for CONTAINERD_VERSION in ${CONTAINERD_VERSIONS}; do
```

Comment:
can we move `CONTAINERD_DOWNLOAD_URL_BASE` down here now that it's not used until `downloadContainerd` is called? It had to be at the top of the file before because `CONTAINERD_DOWNLOAD_URL` was set at the top of the sourced file

Reply:
done

Reply:
I'm being picky but the var assignment doesn't need to be inside the for loop since it's static

Reply:
done